models/br_mf_divida_ativa/code/divida_ativa.py

```
@@ -0,0 +1,371 @@
+                r.raise_for_status()
+                with open(dest, "wb") as fh:
+                    for block in r.iter_content(chunk_size=1 << 20):
+                        fh.write(block)
+            log.info(
+                "downloaded %s -> %s (%.1f MB)",
+                url,
+                dest,
+                dest.stat().st_size / 1e6,
+            )
+            return dest
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)